lib/passageidentity/user_api.rb

```diff
@@ -1,6 +1,9 @@
-require_relative "client"
+# frozen_string_literal: true
+
+require_relative 'client'
 
 module Passage
+  # The UserAPI class provides methods for interacting with Passage Users
```

Comment:
the auth docstring has more information about specifically what it does while this one does not. perhaps a bit of elaboration would be helpful here

Reply:
I was actually simplifying the others based on your other comments. These are autogenerated. If we want to expand on this it may be nice to define the same comments across all repos.